src/index.tsx

```diff
@@ -13,15 +13,34 @@ import SearchHistoryContextProvider from './search-history';
 import TicketContextProvider from './TicketContext';
 import RemoteConfigContextProvider from './RemoteConfigContext';
 import {loadLocalConfig} from './local-config';
-import Bugsnag from '@bugsnag/react-native';
+import Bugsnag, {Event} from '@bugsnag/react-native';
 import {setInstallId as setApiInstallId} from './api/client';
 import ErrorBoundary from './error-boundary';
 import {PreferencesContextProvider} from './preferences';
 
 if (!__DEV__) {
   Bugsnag.start();
 } else {
-  Bugsnag.notify = (error) => console.error(error);
+  Bugsnag.notify = function (error, onError) {
+    const event = Event.create(error, true, {} as any, 'notify()', 1, console);
+    let metadata: {[key: string]: any} = {};
+    event.addMetadata = (
+      section: string,
+      values: string | object,
+      values2?: unknown,
+    ) => {
+      if (typeof values === 'string') {
+        metadata[values] = values2;
+      } else {
+        metadata = {
+          ...metadata,
+          ...values,
+        };
+      }
```

Comment:
:o what's going on here?

Reply:
Oh yeah. It's to support overloading functions in the simplest way 😬